### ISSUE TITLE
/bundles/{identifier}/all does not return any results

### ISSUE THREAD

**Issue by al-niessner** [NASA-PDS/registry-api#118]
## 🐛 Describe the bug

```
$ curl --location --request GET 'http://localhost:8080/bundles/urn:nasa:pds:insight_rad::2.1/all?fields=lidvid' --header 'Accept: application/kvp+json' | jq
  % Total    % Received % Xferd  Average Speed   Time    Time     Time  Current
                                 Dload  Upload   Total   Spent    Left  Speed
100   107  100   107    0     0   7133      0 --:--:-- --:--:-- --:--:--  7133
{
  "summary": {
    "q": "",
    "hits": 0,
    "took": 7,
    "start": 0,
    "limit": 100,
    "sort": [],
    "properties": []
  },
  "data": []
}
```

and

```
$ curl --location --request GET 'http://localhost:8080/bundles/urn:nasa:pds:insight_rad/all?fields=lidvid' --header 'Accept: application/kvp+json' | jq
  % Total    % Received % Xferd  Average Speed   Time    Time     Time  Current
                                 Dload  Upload   Total   Spent    Left  Speed
100   107  100   107    0     0   7642      0 --:--:-- --:--:-- --:--:--  7642
{
  "summary": {
    "q": "",
    "hits": 0,
    "took": 7,
    "start": 0,
    "limit": 100,
    "sort": [],
    "properties": []
  },
  "data": []
}
```

BUT

```
$ curl --location --request GET 'http://localhost:8080/bundles/urn:nasa:pds:insight_rad/latest?fields=lidvid' --header 'Accept: application/kvp+json' | jq
  % Total    % Received % Xferd  Average Speed   Time    Time     Time  Current
                                 Dload  Upload   Total   Spent    Left  Speed
100    46  100    46    0     0   2090      0 --:--:-- --:--:-- --:--:--  2190
{
  "lidvid": "urn:nasa:pds:insight_rad::2.1"
}
```

## 📜 To Reproduce
How to reproduce is in the curls above

## 🕵️ Expected behavior
All and latest should match with the test DB.

---

## 🦄 Related requirements
#106

**Comment by al-niessner** [comment]
@tloubrieu-jpl This was fixed with #112

**Comment by al-niessner** [comment]
fixed by NASA-PDS/registry-mgr#50
